_includes/contact.html

```diff
@@ -0,0 +1,13 @@
+<form action="https://usebasin.com/f/9cbe63444dee" method="POST">
+    <div class="form-input">
+        <label for="email-address">Email Address<abbr title="required" class="usa-hint usa-hint--required">*</abbr></label>
+        <input type="email" id="email" name="email-address" required>
+    </div>
+    <div class="form-input">
+        <label for="text-area">Subject<abbr title="required" class="usa-hint usa-hint--required">*</abbr></label>
+        <textarea id="body" name="text-area"></textarea>
```

Comment:
@jaube05 the label `for=<value>` references the id of input and textarea and not the name. I think you just need to add the id with the same value as the name or modify it to match.